### ISSUE TITLE
City Siege - queing into multiple instances

### ISSUE THREAD

**Issue by M0rw47h** [WarEmu/WarBugs#18083]

**Comment by M0rw47h** [comment]
Made another test and added some evidence.
Joined City Siege, did one fight on destro as BG and crashed my client (12:20), then logged on my SW and qued for Altdorf, got a pop (12:24). Allowed me to play 2nd instance.

**Comment by M0rw47h** [comment]
Another test, to confirm, if it's possible to get rewards from multiple City Sieges happening at once. Yes, it's possible.

Joined City Siege on WP, did one fight and crashed client. Then, I've logged on SW, joined City Siege and instantly crashed client. 
With two different characters set in two different City Siege instance, I was able to play two different instances at same time. I've went back on WP to my CS1 and played ST1 and got rewards for it. After that, I've crashed my client and played all 3 stages on SW in CS2. Clearly needs fast fixing.

Some evidence:
![Nowy projekt (4)](https://user-images.githubusercontent.com/35219756/99549601-1e10b780-29ba-11eb-946c-15477fe186d0.png)
![Nowy projekt (3)](https://user-images.githubusercontent.com/35219756/99549554-118c5f00-29ba-11eb-81b3-707709fb2788.png)
